application/libraries/Api/Command/V1/Transformer/Input/TransformerInputThemeSettings.php

```
@@ -3,28 +3,50 @@
-                'type' => 'string',
-                'required'
+        $entity = App()->request->getParam('_entity');
+        if (isset($entity) && $entity === 'survey-detail') {
+            $surveyId = App()->request->getParam('_id');
+            if (isset($surveyId)) {
+                $surveyId = (int)$surveyId;
+                $templateConfiguration = Template::getTemplateConfiguration(null, $surveyId, null, true);
+                $optionAttributes = TemplateManifest::getOptionAttributes($templateConfiguration->path);
+                $dynamicDataMap = [];
+                if ($optionAttributes) {
+                    foreach ($optionAttributes as $key => $attribute) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$attribute'. (undefined)

(UnusedLocalVariable)